gems/pending/util/miq-password.rb

```diff
@@ -123,7 +123,22 @@ def self.clear_keys
   end
 
   def self.v2_key
-    @v2_key ||= ez_load("#{key_root}/v2_key")
+    @v2_key ||= begin
+      key_file = File.expand_path("v2_key", key_root)
+      if File.exists?(key_file)
+        ez_load(key_file)
+      else
+        msg = <<-EOS
+#{key_file} doesn't exist!
+If you're not a developer, please generate your v2_key using 'appliance_console_cli --key'.
```

Comment:
Yeah, I just threw some text in there. It's more about if we want to do things this way or via #4082 

I can clean up the text if this is the way we want to do it.